lib/irb/command/irb_info.rb

```diff
@@ -13,7 +13,10 @@ def execute
         str += "IRB version: #{IRB.version}\n"
         str += "InputMethod: #{IRB.CurrentContext.io.inspect}\n"
         str += "Completion: #{IRB.CurrentContext.io.respond_to?(:completion_info) ? IRB.CurrentContext.io.completion_info : 'off'}\n"
-        str += ".irbrc path: #{IRB.rc_file}\n" if File.exist?(IRB.rc_file)
+        rc_files = IRB.rc_files.each_with_object([]) do |rc, files|
+          files << rc if File.exist?(rc)
+        end
```

Comment:
I think this can be simply written with `select`
```ruby
IRB.rc_files.select { |rc| File.exist?(rc) }
```

Reply:
Thank you!